cypress/integration/home_page_spec.js

```diff
@@ -2,5 +2,9 @@ describe('Deposits Page', () => {
     it('successfully loads', () => {
         cy.visit('http://localhost:9292/deposits')
         cy.contains('Deposits')
+<<<<<<< HEAD
```

Comment:
These are conflict markers, they should be removed when resolving conflicts

Reply:
okay, so does that mean i remove the indicated lines?

Reply:
Yes, that is how to resolve some conflicts. Next time you get a conflict ask for video help if you not sure what to do 😄

Reply:
sawa